test/config-next/remoteva-b.json

```diff
@@ -36,7 +36,9 @@
 		"accountURIPrefixes": [
 			"http://boulder.service.consul:4000/acme/reg/",
 			"http://boulder.service.consul:4001/acme/acct/"
-		]
+		],
+		"cluster": "development",
+		"rir": "ARIN"
```

Comment:
nit: this would put both CI remotevas in the same place, which would be a violation of the requirements. We should have them pretend to be in different places.